source/js/RecipeCard.js

```diff
@@ -0,0 +1,350 @@
+class RecipeCard extends HTMLElement {
+  constructor() {
```

Comment:
Codacy has a fix for the issue: [Missing space before function parentheses.](https://app.codacy.com/gh/cse110-fa21-group27/cse110-fa21-group27/pullRequest?prid=8437408)
```suggestion
  constructor () {
```